Bottom: 8cccb5592f85b5dd986b7edf823a3199f4aed6ba
Top:    64125505b2ccb07004f5883a18f62132b8a0cbcd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 23:40:40 +0100

Refresh of hid-lesstif-remove-unused-vari

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 776a345..cdaa7b0 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3039,8 +3039,6 @@ lesstif_destroy_gc (hidGC gc)
 static void
 lesstif_use_mask (int use_it)
 {
-  static Window old;
-
   if ((TEST_FLAG (THINDRAWFLAG, PCB) || TEST_FLAG(THINDRAWPOLYFLAG, PCB)) &&
       !use_xrender)
     use_it = 0;
